config/sites/registrar.uiowa.edu/.htaccess

```diff
@@ -19,6 +19,6 @@ SetHandler Drupal_Security_Do_Not_Remove_See_SA_2006_006
 </Files>
 
 # If we know how to do it safely, disable the PHP engine entirely.
-<IfModule mod_php7.c>
+<IfModule mod_php.c>
```

Comment:
Was this an automatic update? @bspeare, @joewhitsitt?

I'm wondering because it seems like we would need to do this for all the other config folders too.

Reply:
i feel like this has been updating after config exports. we have a mix of both versions as we have worked on them.